### DISCUSSION TITLE
検索でグループ検索"()"は使えますか？

### DISCUSSION THREAD

**Discussion by didvc** [increments/qiita-discussions#367] [ヘルプ] [answered]
具体例といたしまして、以下のようなクエリを実行したいのですが

```
画像 類似 stocks:>3 created:>2021-2-01 ( tag:Kaggle OR tag:機械学習 OR tag:Python OR tag:画像処理 )
```

かっこ部分のグルーピングが無視されているようで、検索結果数を見る限り機能していないようです。

上記のような検索演算子はQiitaに用意されていますか？

ご教授よろしくお願いします。

**Comment by wataru86** [comment, marked as answer]
@yuis-ice
ご意見ありがとうございます🙏
申し訳ありませんが、上記のようなグループ検索機能はありません 🙇 
検索に関する他のご意見もいただいており、改善したいと思っておりますので、今回いただいたご意見も合わせて検討します！💪

**Comment by wataru86** [comment]
@yuis-ice
お待たせしました🙇 複数のタグのORを `,` 区切りで記述できるようになりました。
以下のクエリをお試しください。

```
画像 類似 stocks:>3 created:>2021-2-01 tag:Kaggle,機械学習,Python,画像処理
```

[詳細はこちら](https://help.qiita.com/ja/articles/qiita-search-options)

引き続き検索機能の改善に努めてまいりますので、今後ともQiitaをご愛顧いただけますと幸いです✨
